.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: php
+
+php:
+  - 7.2
+
+sudo: false
+
+install:
+  - travis_retry composer install --dev
+
+script:
+  - composer validate --no-interaction
+  - make phpunit
+  - make cs
+  - make phpmd
+
+after_success:
```

Comment:
You can drop the `after_success:` configuration if there is nothing to be done. In other projects this is used to upload coverage data to scrutinizer, but here this is not done.